lib/galaxy/web/proxy/__init__.py

```diff
@@ -106,6 +106,7 @@ def launch_proxy_command(self, config):
             "--sessions", config.proxy_session_map,
             "--ip", config.dynamic_proxy_bind_ip,
             "--port", str(config.dynamic_proxy_bind_port),
+            "--cookie", "galaxysession",
```

Comment:
This must have been whilst debugging the issues with proxy_ipc. Having [seen](https://docs.galaxyproject.org/en/master/admin/interactive_environments.html#setting-up-the-proxy) and noting that it's not present when galaxy launched the proxy.